### ISSUE TITLE
Voicing reads out wrong voice as you switch

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/friction#268]
**Test device**
Dell
**Operating System**
Win 11
**Browser**
Firefox
**Problem description**
For https://github.com/phetsims/qa/issues/717.
When switching between the voicing options, the voice that is read out seems to be the one you were on previously, not the new one you switched to.

**Visuals**

https://user-images.githubusercontent.com/41024075/139108522-3d728c7a-42ee-4817-af05-e09ff7e5ebd3.MOV

Troubleshooting information:
<details

!!!!! DO NOT EDIT !!!!!
Name: ‪Friction‬
URL: https://phet-dev.colorado.edu/html/friction/1.6.0-dev.19/phet/friction_all_phet.html
Version: 1.6.0-dev.19 2021-10-08 22:59:15 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:93.0) Gecko/20100101 Firefox/93.0
Language: en-US
Window: 1280x667
Pixel Ratio: 1.5/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0
Vendor: Mozilla (ANGLE (Intel(R) HD Graphics Direct3D11 vs_5_0 ps_5_0))
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by zepumph** [comment]
@jessegreenberg and I worked on this, and fixed it in master. @KatieWoe can you please test on master.

**Comment by KatieWoe** [comment]
On Firefox, the whole sim freezes when you turn on voicing

**Comment by jessegreenberg** [comment]
Thanks @KatieWoe, looks like an unrelated issue with Voicing in Firefox after https://github.com/phetsims/scenery/issues/1288

**Comment by jessegreenberg** [comment]
This should be fixed now. @KatieWoe can you please try again to see if this is fixed?

**Comment by KatieWoe** [comment]
Looks fixed on master

**Comment by zepumph** [comment]
Thanks!

**Comment by Nancy-Salpepi** [comment]
I am seeing this issue again in  https://github.com/phetsims/qa/issues/791 with iOS 15.4 + Safari.

Ignore my big hand in this video!

https://user-images.githubusercontent.com/87318828/161594340-933a8919-2314-4573-a18f-8b967802201b.mov

**Comment by jessegreenberg** [comment]
Thanks @Nancy-Salpepi - what you are showing looks a bit different than the original post. The original issue was related to the wrong voice right after making a ComboBox selection but this appears to be the entirely wrong voice for all subsequent interactions. Moving this comment to utterance-queue.
